src/main/java/dinkplugin/notifiers/LootNotifier.java

```diff
@@ -57,6 +62,28 @@ public void onLootReceived(LootReceived lootReceived) {
         }
     }
 
+    public void onWidgetLoaded(WidgetLoaded event) {
+        // special case: runelite client & loot tracker do not handle unsired loot at the time of writing
+        if (event.getGroupId() == WidgetID.DIALOG_SPRITE_GROUP_ID && isEnabled()) {
+            Widget spriteWidget = client.getWidget(WidgetInfo.DIALOG_SPRITE);
+            if (spriteWidget == null || spriteWidget.getItemId() < 0) {
+                spriteWidget = client.getWidget(WidgetInfo.DIALOG_SPRITE_SPRITE);
```

Comment:
Is this something we'd like to add some logging to so we could potentially simplify this in the future?

Reply:
Good idea - added logging to determine which widget was used to get the item info & some logging in the case where no item was found (to have some leads to investigate in identifying the item) in https://github.com/pajlads/DinkPlugin/pull/89/commits/f280472bea726e9a866864109b9492a78879c5f6